packages/ui/src/BooleanField.tsx

```diff
@@ -1,19 +1,21 @@
-import React, {ReactElement, useContext, useRef} from "react";
+import React, {ReactElement, useRef} from "react";
 import {Animated, TouchableWithoutFeedback, View} from "react-native";
 
 import {BooleanFieldProps} from "./Common";
+import {FieldHelperText, FieldTitle} from "./FieldElements";
 import {Text} from "./Text";
-import {ThemeContext} from "./Theme";
+import {useTheme} from "./Theme";
 
 export const BooleanField = ({
-  label,
+  title,
   variant,
   value,
   onChange,
   interaction = true,
```

Comment:
Is there a reason for using "interaction" instead of "disabled" ? I think we should use disabled here if not.

Reply:
Hmmmm no, I don't think so! I'm going to change it. I thought we weren't allowing disabled for boolean since there was no disabled prop, but looking at Figma we do for boolean, don't for checkbox. I'll change.